stdlib/LinearAlgebra/test/matmul.jl

```diff
@@ -152,7 +152,7 @@ end
     for vf in (copy(vvf), view(vvf, 1:3)), C in (copy(CC), view(CC, 1:3, 1:3))
         @test mul!(C, vf, transpose(vf)) == vf*vf'
         C .= C0 = rand(eltype(C), size(C))
-        @test mul!(C, vf, transpose(vf), 2, 3) == 2vf*vf' .+ 3C0
+        @test mul!(C, vf, transpose(vf), 2, 3) ≈ 2vf*vf' .+ 3C0
```

Comment:
I'm wondering if this was exact equality for a reason, cc @tkf

Reply:
The difference is `1e-15` in one of the elements. 3 of these tests fail with MKL.

Reply:
In the test above, I think the idea is that it should be the exact same code path and therefore exact equality. It wasn't obvious to me that this test should take the same path for right and left hand side but wondered if that was your intention. If not then this PR should be fine.

Reply:
Yeah, sorry, I was sloppy. This change LGTM too.